test/jdk/java/nio/jni/NewDirectByteBuffer.java

```diff
@@ -75,6 +101,7 @@ public static void main(String[] args) {
     }
 
     // See libNewDirectByteBuffer.c for implementations.
-    private static native ByteBuffer allocBigBuffer(long size);
-    private static native long getLongCapacity(ByteBuffer buf);
+    private static native ByteBuffer newDirectByteBuffer(long size);
+    private static native long getDirectBufferCapacity(ByteBuffer buf);
+    private static native void freeDirectBufferMemory(ByteBuffer buf);
```

Comment:
Naming nit: ByteBuffer versus Buffer

Reply:
Fixed in 95fd6652148e1b740af599081b3614c54656c8b4 (overlooked in previous commit).